.gitignore

```diff
@@ -1,2 +1,23 @@
 .DS_Store
 /target
+
+# Generated by Cargo
+# will have compiled files and executables
+debug/
+target/
+
+# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
+# More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
+Cargo.lock
+
+# These are backup files generated by rustfmt
+**/*.rs.bk
+
+# MSVC Windows builds of rustc generate these, which store debugging information
+*.pdb
+
+# VSCode
+.vscode/
```

Comment:
Please use `/.vscode/` for directories in the project root.

Reply:
Done.